lib/components/artist/artworks/grid.js

```diff
@@ -137,13 +139,15 @@ class ArtworksGrid extends React.Component {
           }
         }
 
-        const section = sectionedArtworks[sectionIndex]
-        section.push(artwork)
+        if (sectionIndex) {
```

Comment:
you'll see this a few times. 

So, `let sectionIndex: ?number = null` declares the `sectionIndex` to be a nullable number. Then we find the lowest value it should be:
```js
       for (let j = 0; j < sectionRatioSums.length; j++) {
           const ratioSum = sectionRatioSums[j]
           if (ratioSum < lowestRatioSum) {
             sectionIndex = j
             lowestRatioSum = ratioSum
            }
          }
```

and from this point, we're sure it's something. _we know this_ but the type system doesn't, because assignment is hidden behind that if statement `if (ratioSum < lowestRatioSum)` making it not comprehensively set. 

This means when we get to this line (142) sectionIndex is still `nullable number` not a guaranteed number. So, we do an if statement to force it into always existing. This is good, because I think `sectionedArtworks[null]` would cause pretty unexpected results.